### PR TITLE
completes first quiz

### PR DESCRIPTION
**Pull request by herrerake**
Completeness:  4
comfort: 3
Comment: I'm not going to lie I am not all the way comfortable with JS. I did have trouble with chapter 10 in the pre-work and hit a wall there. I still need to finish pre-work. Git, git-hub, html, and css I am pretty comfortable with.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ answer goes here
 Write a for loop such that you say hello(using `console.log`) to each name in the `names` array
 
 ```js
-answer goes here
+Not comfortable answering this question.
 ```
```
Comment (superbuggy) [inline readme.md:54]: no problem! let's meet up tomorrow at some point

File: readme.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ answer goes here
 Access the value `"Alfred"` out of the `clown` object
 
 ```js
-answer goes here
+Not comfortable answering this question.
 ```
 
 ---
```
Comment (superbuggy) [inline readme.md:66]: see above comment

File: readme.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ answer goes here
 What is Git, what problem does it solve? What is the difference between `git` and github?
 
 ```
-answer goes here
+Git solves the problem of version control. It helps organize and keep a "file cabinet" of all the files of a given project. Git lives on your computer and github is a online server used to collaborate with different git and github users.
 
 ```
```
Comment (superbuggy) [inline readme.md:77]: 👍  Another crucial feature of GitHub is that it hosts repositories, but you basically imply that

File: readme.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ answer goes here
 What is the difference between a fork and a clone?
 
 ```
-answer goes here
+A fork takes from another users repository and makes it your own. A clone takes from your online repository and "clones" it to your own local repository.
 
 ```
```
Comment (superbuggy) [inline readme.md:88]: really forking is creating a copy on your own account, but that's probably what you meant

File: readme.md [hunk 5]
```diff
@@ -145,5 +145,5 @@ answer goes here
 Write a CSS selector-rule that will select and apply styling to every `<li>` inside of a `<ul>`:
 
 ```
-answer goes here
+li > ul
 ```
```
Comment (superbuggy) [inline readme.md:149]: super close! it's `ul > li` (you could also answer `ul li`)

The pattern is 

``` css
parent child {
  /*CSS Here*/
}
```

File: readme.md [hunk 6]
```diff
@@ -135,7 +135,7 @@ Write a CSS selector-rule that will select and apply styling to an element with
 
 
 ```
-answer goes here
+#dog-resume
 ```
```
Comment (superbuggy) [inline readme.md:140]: nailed it

File: readme.md [hunk 7]
```diff
@@ -106,7 +106,7 @@ answer goes here
 How would you link a css file entitled, `hardstyle.css` in an html file?
 
 ```
-answer goes here
+<link rel="stylesheet" type="text/css" href="hardstyle.css">
 ```
```
Comment (superbuggy) [inline readme.md:111]: 👍

File: readme.md [hunk 8]
```diff
@@ -96,7 +96,7 @@ answer goes here
 What does every HTML file need to be recognized as such? In other words, what do you need in each HTML file for it to be valid HTML?
 
 ```
-answer goes here
+<!DOCTYPE html>
 ```
```
Comment (superbuggy) [inline readme.md:101]: 👍 

also probably need head and body thoough

File: readme.md (hunk 1 above)
Comment (herrerake) [reply readme.md:54]: Greeat, yeah lets do that. Thaanks

**Pull request closed without merging**